langkit/diagnostics.py

```diff
@@ -6,7 +6,9 @@
 import os.path as P
 import sys
 import traceback
-from typing import NoReturn
+from typing import NoReturn, Optional
+
+import liblktlang as L
```

Comment:
This breaks the bootstrap on this commit. Could you move the introduction of the `try/except ImportError` block in this commit?

Reply:
Done